server/controllers/userController.js

```diff
@@ -97,10 +97,41 @@ const register = async (req, res) => {
   }
 };
 
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    // check if user exists
+    const findUser = await findUserByEmail(email);
+
+    if (!findUser) {
```

Comment:
Should include status code in the 400 range for the client here. Might also want to consider returning a UserName / Password not found combined message. A malicious actor could use this error to determine which user names are and are not in our database.